docs/overview.mdx

```diff
@@ -24,9 +24,17 @@ Use it with [Elastic](https://www.elastic.co/) to have a search experience up an
 
 ### Live Demo
 
-Check out the [live demo of Search UI](https://search-ui-stable.netlify.com).
-
-[![Edit search-ui-national-parks-example](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/search-ui-national-parks-example-ts-k6u5iz)
+<iframe
+  src="https://codesandbox.io/embed/github/elastic/search-ui/tree/master/examples/sandbox?autoresize=1&fontsize=12&initialpath=%2Felasticsearch&module=%2Fsrc%2Fpages%2Felasticsearch%2Findex.js&theme=light&view=preview&hidedevtools=1"
```

Comment:
Should this be from "stable" or "master"?

Reply:
Nice catch, should be stable.